Engineering.md

```diff
@@ -1,5 +1,38 @@
 # Engineering Roles
 
+## Junior Engineer
+
+> I am an enthusiastic learner, continually improving my value to an Octopus team.
+
+### I am an egoless, team oriented developer
+
+- Other team members genuinely want me on their team.
+- I happily take responsibility for what I produce.
+- I avoid going dark and I know when to call out for help.
+- I communicate professionally and keep stake holders engaged at all times.
+- I am well aware of all processes of the team and diligently follow them (like maintaining accurate documentation, maintaining tests, and so on).
```

Comment:
`I am well aware of all processes of the team and diligently follow` this language seems intimidating for a junior engineer

Reply:
Good point, I think this could get sent back up to `Engineer` and softened to include words like "learning about"